src/bms/command/minor_command.rs

```diff
@@ -18,12 +24,6 @@ impl ExWavPan {
         (-10000..=10000).contains(&value).then_some(Self(value))
     }
 
-    /// Returns the underlying value.
-    #[must_use]
-    pub const fn value(self) -> i64 {
-        self.0
-    }
```

Comment:
I think `value` methods are still useful. It can extract the contained by one shot.